src/dom.js

```diff
@@ -284,7 +284,7 @@ class dom {
 		var limit = event.currentTarget.parentNode;
 		var defFns = [];
 
-		while (currElement && currElement !== limit && !event.stopped) {
+		while (currElement && currElement !== limit && !event.stopped && !currElement.disabled) {
```

Comment:
Hi, @mairatma.

We need to discuss this implementation. I'm not sure if I understood this feature clearly.

Besides that, I realize that browsers have different behaviours for disabled elements.

We can discuss it tomorrow on Slack.